README.md

```diff
@@ -1,2 +1,28 @@
-# New-York-Times
 article from the New York Times project 
+
+
+New York Times Page
+
+this is a project about new york times clone using HTML and CSS. This one page project for mircoverse in first week – completed 19 feb – sets out to deliver on one article for new york times about space.
+
+
+
+
+
+
+
+Built With
+
+HTML
+
+CSS
+
+flexbox
+
+Live Demo
+
+https://amalcxc.github.io/New-York-Times/
+
+Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
## The `README` can further get improved to look more presentable.
The `README` has some useful information on what the project is all about. In my view, I think we can make it look more professional to have people understand in-depth what the project is all about in one place. With that, I would suggest we use this [template](https://github.com/microverseinc/readme-template) as an inspiration in adding some crucial information like a screenshot of the page, the author's (you in this case) social links, how to well clone and run the project locally, and have an actual live link this [one](https://raw.githack.com/Amalcxc/New-York-Times/one-page-article/newyork.html) of the actual live page. Right now, this is where users are getting redirected. 👇 

![readme](https://user-images.githubusercontent.com/22345961/108566892-35907100-7318-11eb-826b-5079cf9b74d4.png)

That is more or like the live version of the `README` itself.  I have a suggestion that you can use to generate a live version of the page using this [tool](https://raw.githack.com/). You are welcome to use [this](https://thecompetentdev.com/weeklyjstips/tips/7_serve_files_from_github/) tutorial on how you can go about this as well. What do you think? (note: this is just but one of the ways you can achieve this. You don't need to follow this solution, thank you.)
##
### Here is a summary of things you might want to focus on while working on these suggestions so as to have a `README` that is more descriptive as indicated in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/).
- [ ] Have a live link to the page.
- [ ] Have a screenshot of the page.
- [ ] Have your social links in the author's section of the `README`.
- [ ] Have instructions on how to well clone and run the project locally.
- [ ] You might want to check on grammatical issues, they are bound to appear from time to time when working with documentations.